File: Libraries/WowAutoApp.Core/Domain/Enums/EmploymentStatusType.cs
using System.ComponentModel;

namespace WowAutoApp.Core.Domain.Enums
{
    /// <summary>
    /// Credit application residence owner type
    /// </summary>
    public enum EmploymentStatusType
    {
        [Description("Self Employed")]
        SelfEmployed = 0,
        [Description("Contract")]
        Contract = 1,
        [Description("Retired")]
        Retired = 2,
        [Description("Unemployed")]
        Unemployed = 3,
        [Description("Business Owner")]
        BusinessOwner = 4
    }
}

File: WowAutoApp.Web.Client/src/app/core/enums/EmploymentStatusType.ts
export enum EmploymentStatusType {
    SelfEmployed = 0,
    Contract  = 1,
    Retired = 2,
    Unemployed = 3,
    BusinessOwner = 4
}

export namespace EmploymentStatusType {

    export function values() {
        return Object.keys(EmploymentStatusType).filter(
            (type) => isNaN(<any>type) && type !== 'values'
        );
    }
}   